mscore/scoretab.cpp

```diff
@@ -289,7 +289,7 @@ void ScoreTab::updateExcerpts()
             }
       else {
             tab2->setVisible(false);
-            setCurrent(0);
+            setExcerpt(0);
```

Comment:
is this really needed? It is done 2 lines later again

Reply:
It is indeed done three lines later but with blocked signals. This leads to not emitting `currentScoreViewChanged` signal and thus not setting a proper current score view. This, in turn, leads to possible crashes like the one described in the [original issue](https://musescore.org/en/node/275808). I am not sure though why that latter call with blocked signals is needed.

Reply:
In fact this signals blocking can even be a reason for [another crash](https://musescore.org/en/node/276628). Still this needs to be checked again.